src/ripple/app/tx/impl/NFTokenAcceptOffer.cpp

```diff
@@ -206,15 +217,39 @@ NFTokenAcceptOffer::preclaim(PreclaimContext const& ctx)
 
         // The account offering to buy must have funds:
         auto const needed = so->at(sfAmount);
```

Comment:
Just curious why the comment above says `The account offering to buy must have funds`. Isn't this for sell offer?

Reply:
Nevermind, if i understand correctly, it check if the buyer who accepts a sell offer has enough money.